src/features/game/expansion/Land.tsx

```diff
@@ -118,17 +120,31 @@ const Expansion: React.FC<ExpansionProps & { expansionIndex: number }> = ({
             </MapPlacement>
           );
         })}
+
+      {stones &&
+        getKeys(stones).map((index) => {
+          const { x, y, width, height } = stones[index];
+
+          return (
+            <MapPlacement
+              key={`${createdAt}-stone-${index}`}
+              x={x + xOffset}
+              y={y + yOffset}
+              height={height}
+              width={width}
+            >
+              <div style={{ background: "black" }} className="w-full h-full" />
```

Comment:
A stone is currently represented by a black div as stones have not been implemented yet.